include/mbedtls/camellia.h

```diff
@@ -175,7 +181,7 @@ int mbedtls_camellia_crypt_cbc( mbedtls_camellia_context *ctx,
  * \param output   buffer holding the output data
  *
  * \return         0 if successful, or
- *                 MBEDTLS_ERR_CAMELLIA_INVALID_INPUT_LENGTH
```

Comment:
Should this keep saying `MBEDTLS_ERR_CAMELLIA_INVALID_INPUT_LENGTH`? I don't think we want to switch `MBEDTLS_ERR_CAMELLIA_INVALID_INPUT_LENGTH` to `MBEDTLS_ERR_CAMELLIA_BAD_INPUT_DATA`.

Reply:
This function actually never returns `MBEDTLS_ERR_CAMELLIA_INVALID_INPUT_LENGTH`, as will be documented in https://github.com/ARMmbed/mbedtls/pull/2281/files#diff-a075027398f751a1024274ad57041090R206.

Reply:
Why change the documentation in this PR at all if we change it later to be correct? Seems odd to change it from one incorrect thing to anther incorrect thing.

Reply:
The change was a simple text replacement of error codes without evaluating what the implementation does. In retrospect, it wasn't necessary. Looking at the implementation, though, it doesn't make sense to revert it now.